misc/docker/base_images/debian_7_wheezy_32bit/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM obspy/base-images:debian_7_wheezy_32bit
+
+MAINTAINER Tobias Megies
+
+# Set the env variables to non-interactive
+ENV DEBIAN_FRONTEND=noninteractive DEBIAN_PRIORITY=critical DEBCONF_NOWARNINGS=yes
+
+# install packages to install obspy and build deb packages
+RUN apt-get update && apt-get upgrade -y || true
+RUN apt-get update && apt-get -y --no-install-recommends install \
+    debhelper \
+    devscripts \
+    equivs \
+    fakeroot \
+    gcc \
+    gfortran \
+    git \
+    help2man \
+    lintian \
+    locales \
+    lsb-release \
+    python \
+    python-decorator \
+    python-dev \
+    python-imaging \
+    python-lxml \
+    python-matplotlib \
+    python-mock \
+    python-mpltoolkits.basemap \
+    python-mpltoolkits.basemap-data \
+    python-nose \
+    python-numpy \
+    python-pip \
+    python-requests \
+    python-scipy \
+    python-setuptools \
+    python-sqlalchemy \
+    python-suds \
+    python-support \
+    python-tornado \
+    quilt \
+    ttf-bitstream-vera \
+    vim \
```

Comment:
I probably had it in there for manual debugging inside containers.. not sure, think i'll leave it in, it's not very big (recommends are not installed).